src/Pages/Auth/Components/Buttons.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import { FaCalendar } from 'react-icons/fa';
+
+const Buttons = props => {
+  const { format } = props;
+  return format === '로그인' ||
+    format === '동의하고 회원가입' ||
+    format === '다른 방법으로 시작하기' ? (
+    <Button>{format}</Button>
+  ) : (
+    <Kakao>
+      <FaCalendar />
+      &nbsp;&nbsp;카카오로 3초 만에 시작하기
+    </Kakao>
+  );
+};
```

Comment:
```suggestion
const Buttons = ({ format }) => {
  const notKaKao = format === '로그인' ||  format === '동의하고 회원가입' || format === '다른 방법으로 시작하기;
  
  return notKakao ? (
  <Button>{format}</Button>
  )
  : (
    <Kakao>
      <FaCalendar />
      &nbsp;&nbsp;카카오로 3초 만에 시작하기
    </Kakao>
  )
};
```

- props 객체 구조 분해 할당 적용
- 길어지는 조건문 변수화
- 두 가지 적용하여 위와 같이 작성할 수도 있겠네요! 참고해주시기 바랍니다. 
- (notKakao 변수명은 좀 더 고민해주세요...)

Reply:
오 준식님 감사합니다!!! 이런 식으로 구조 분해 할당할 수 있다는 걸 처음 알았어요!! 코드도 더 좋은 방식으로 개선해보겠습니다☺️